settings.gradle

```diff
@@ -12,18 +12,21 @@ pluginManagement {
   repositories {
     mavenLocal()
     gradlePluginPortal()
-    mavenCentral()
     maven {
-      name = "starterBootPkgs"
-      url = uri("https://maven.pkg.github.com/department-of-veterans-affairs/lighthouse-di-starter-boot")
-      credentials(HttpHeaderCredentials) {
-        name = "Authorization"
-        value = "Bearer " + (System.getenv("GITHUB_ACCESS_TOKEN") ?: GITHUB_ACCESS_TOKEN)
-      }
-      authentication {
-        register("header", HttpHeaderAuthentication)
-      }
+      url "https://repo1.maven.org/maven2/"
```

Comment:
I had to use this alternative maven repo when `mavenCentral` was down. Adding it here so others don't waste time trying to fix it.